orc8r/cloud/go/services/metricsd/exporters/remote_exporter.go

```diff
@@ -40,3 +40,27 @@ func (r *remoteExporter) getExporterClient() (protos.MetricsExporterClient, erro
 	}
 	return protos.NewMetricsExporterClient(conn), nil
 }
+
+type TestMetricExporter struct {
```

Comment:
Nit: let's move this to a test (or test helpers) pkg. Generally speaking, don't want to put test-specific code in the same pkg as "normal" code, to avoid confusion

(Of course sometimes there's no other option, or it makes the most sense. But in this case I think we can drop it somewhere else)

Reply:
i just replicated this in the two places it was used in `servicer_southbound_test.go` and `servicer_protected_test.go`